docs/contributor/draft/rate-limiting-poc/06-sum-up-rate-limiting.md

```diff
@@ -0,0 +1,36 @@
+# Summary of Rate Limiting POC
+
+Istio supports rate limiting by using EnvoyFilter configurations to handle the rate limiting in the Envoy proxies. This can be set to local or global rate limiting. Local rate limiting does not require communication with a rate limit service, and each Envoy instance acts independently. Global rate limiting requires gRPC communication with a rate limit service, for example [envoy ratelimit service](https://github.com/envoyproxy/ratelimit).
+
+Envoy rate limit service needs to use Redis or Memcached backend.
+
+# Memcached
+
+Pros:
+- Mature project, used in production by many
+- Easy to deploy in a Kyma cluster
+Cons:
```

Comment:
Did we check if there is a managed BTP offering? If not I would also list this as a con, since this will increase operational effort.

Reply:
I just checked that, there is no memcached on BTP.